components/event-gateway/pkg/nats/authenticator.go

```diff
@@ -160,6 +161,9 @@ func habNatsUser() *natsd.User {
 				Allow: []string{
 					"_INBOX.>",
 					"_STAN.acks.>",
+					"_HB.>",
+					"_STAN.discover.event-service.*",
+					">",
```

Comment:
This one is a little scary but I'm okay to move forward with it for now. We should at least put a card in one of our epics to revisit this.

Reply:
yep, totally agree